modules/@apostrophecms/ui/ui/apos/mixins/AposCellMixin.js

```diff
@@ -12,9 +12,7 @@ export default {
     },
     draft: {
       type: Object,
-      default() {
-        return null;
-      }
+      required: true
```

Comment:
Since `draft` is now required, we have no need for `item` as a prop. The manager is doing logic that could be done in the mixin instead. Contain that and let the manager only pass in two clear things.